### PR TITLE
[OperationalInsights] Add stable version for query packs 

### PR DESCRIPTION
**Pull request by dabenhamMic**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by dabenhamMic** [comment]
@akning-ms can you please help me with the errors in this pr?

**Comment by akning-ms** [comment]
@jianyexi to help take a Linter error as there is exception.
For avocado, @dabenhamMic , you may need to add your latest API in your default readme tag. refer to: https://github.com/Azure/avocado/blob/master/README.md#NOT_LATEST_API_VERSION_IN_DEFAULT_TAG

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPackQueries.json [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
```
Comment (jianyexi) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPackQueries.json:30]:
```suggestion
      "description": "Azure Active Directory OAuth2 Flow",
```
this is different with other swaggers in the tag package-2022-02

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPacks.json [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
```
Comment (jianyexi) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPacks.json:30]:
```suggestion
      "description": "Azure Active Directory OAuth2 Flow",
```
same as above

**Comment by dabenhamMic** [comment]
hi @jianyexi , thank you for the help. Can we proceed with this pr?

**Comment by raych1** [comment]
@msyyc , can you have a look at the python SDK failures?

@dabenhamMic, can you remove `holdon` label if this PR is ready to merge.

**Comment by dabenhamMic** [comment]
@raych1 , tag was removed. 
Is there anything else needed here?

**Comment by msyyc** [comment]
Have fixed python failure

**Comment by dabenhamMic** [comment]
@msyyc , @raych1 can this PR be merged?

**Comment by akning-ms** [comment]
Hi @dabenhamMic, I just noticed you are creating a 2019 stable version, but your latest stable is :2021-06-01. Looks you are just moving a preview version into stable version, right? if so, can you use OpenAPI hub to do it which will make review more easy for us to know what change your PR include: below is guide in ARM review section in first comment
Adding a new API version
-[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

**Comment by dabenhamMic** [comment]
> o stable version, right? if so, can you use OpenAPI hub to do it which will make review more

@akning-ms  This change is not only bumping the API version from package-2021-12-01-preview to package-2022-02,
but also the addition of 2 resources:
1. Microsoft.OperationalInsights/stable/2019-09-01/QueryPackQueries.json
2. Microsoft.OperationalInsights/stable/2019-09-01/QueryPacks.json

haven't found a way to do both actions through the link you sent...

**Comment by akning-ms** [comment]
> ven't found a way to do both actions through the link you sent...

My key question is: beside version change from 2019-09-01-preview to 2019-09-01, what actual change you have made?
as github diff is easy for us to know what change you had, so openapi hub can help you do copy file and change version and adding readme tag for a new version which can save you and us many effort.  if openapi hub can not help you do all, you can make new commit to add it. so if you can confirm no other change for your new version. I can help you merge, but strong recommend you to do it in the future for all new API version

**Review by akning-ms — APPROVED** [review]

**Pull request merged**